python/lsst/validate/drp/checkAstrometryPhotometry.py

```diff
@@ -50,8 +54,8 @@ def getCcdKeyName(dataid):
 
     Notes
     -----
-    Motiviation: Different camera mappings use different keys to indicate 
-      the different amps/ccds in the same exposure.  This function looks 
+    Motiviation: Different camera mappings use different keys to indicate
+      the different amps/ccds in the same exposure.  This function looks
```

Comment:
"Motiviation".

Also, I'd like to highlight the existence of this function as a use-case for having camera-generic data IDs or data ID aliases built into the butler or base-class mapper.  Not that there's a proposal for doing that on the table anywhere right now, but I think we're chipping away at the argument that it's not a problem for different cameras to have totally different names for the same things.